app/src/main/res/values/strings.xml

```diff
@@ -190,10 +190,9 @@
   <string name="map_theme_dark">mapbox://styles/mapbox/traffic-night-v2</string>
   <string name="mapbox_commons_app_token">pk.eyJ1IjoibWFza2FyYXZpdmVrIiwiYSI6ImNqMmxvdzFjMTAwMHYzM283ZWM3eW5tcDAifQ.ib5SZ9EVjwJe6GSKve0bcg</string>
   <string name="_2fa_code">2FA Code</string>
-  <string name="number_of_uploads">My Recent Upload Limit</string>
   <string name="maximum_limit">Maximum Limit</string>
   <string name="maximum_limit_alert">Unable to display more than 500</string>
-  <string name="set_limit">Set Recent Upload Limit</string>
+  <string name="set_limit">Recent upload limit</string>
```

Comment:
I think "Set recent upload limit" title would be better. I agree about unnecessary capitals but I do think we should follow imperative form in preferences. See:
- Use external storage
- Display nearby notification

Reply:
I disagree with imperative form in preferences. I think the two examples you have should keep their verbs as you are toggling wether that behaviour happens. However, "set" is what you're doing right now.

I.e. clicking "display nearby notification" doesn't display the notification but sets whether the app should display the nearby notification. Likewise all the settings have an implied "set" or "set whether the app".

For comparison in android sound settings on my phone there is:
- Phone ringtone (parallels with setting a custom value like upload count)
- Vibrate for calls (parallels with setting whether the app should perform an action like display nearby notification)

Also night mode, default license are currently not.

Reply:
Hmm, from this point of view, you are right. But do you think user will notice this setting is something they can click and change if we say "Recent upload limit" ?

Reply:
I believe so - everything else in settings is something they can click. This design pattern is used in android's own settings and is already true of "default license".

Reply:
Just did a bit more research, Android's documentation states titles should avoid "Generic terms, such as **set**, change, edit, modify, manage, use, select, or choose."

Also you can see none of the screenshots there (which I believe are from the Pixel settings app) have "set", some with options that are similar (as in they are just modifying some integer value)

https://source.android.com/devices/tech/settings/settings-guidelines